apps/settings/js/panels/call/call_forwarding.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Make sure all operations would be executed one by one.
+ *
+ * @module call/call_forwarding
+ */
+define(function(require) {
+  'use strict';
+
+  var CallConstant = require('panels/call/call_constant');
+
+  /**
+   * @class CallForwarding
+   * @returns {CallForwarding}
+   */
+  var CallForwarding = function() {
+    this._cardIndex = 0;
```

Comment:
As there will be instance for each sim, I would suggest to take the card index in the construct then we don't need to pass it when calling the functions.